bin/direxec.sh

```diff
@@ -1,21 +0,0 @@
-#!/bin/bash
-# Thanks Camptocamp for the idea!
-# http://www.camptocamp.com/en/actualite/flexible-docker-entrypoints-scripts/
-set -e
-
-mode=$(basename $0 .sh)
-
-for dir in custom common; do
-    dir=/opt/odoo/$dir/$mode.d
-    if [ -d $dir ]; then
-        log INFO Executing contents of $dir
-        run-parts --exit-on-error $dir
-    else
-        log WARNING Skipping execution of $dir: not found
-    fi
-done
-
-if [ -n "$1" ]; then
-    set -x
-    exec "$@"
```

Comment:
> makes the subprocess here take the PID of the parent process 
Hmmm so subprocess `shell=False` does this, but not `shell=True`. Looking at the `os.system` docs it explicitly says that it is running in a subshell, which I think would mean the same PID. I'm not exactly sure how to test though...